Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/Pointers/TeleportPointer.cs

```diff
@@ -224,7 +225,7 @@ public override void OnPostRaycast()
                             }
 
                             // Only add the distance between the start point and the hit
-                            clearWorldLength += Vector3.Distance(Result.StartPoint, Result.StartPoint);
+                            clearWorldLength += Vector3.Distance(Result.StartPoint, Result.Details.Point);
```

Comment:
@davidkline-ms this distance calculation is only for properly clamping the line renderer to properly render to the end of the parabola. It is not used in calculating the validity of the teleport result and is outside the scope of this PR.